temfpy/uncertainty_quantification.py

```diff
@@ -5,15 +5,50 @@
 
 """
 import numpy as np
-import math
 
 
 def borehole(x):
     r"""Borehole function.
 
-    The Borehole function models water flow through a borehole. Its simplicity and quick
-    evaluation makes it a commonly used function for testing a wide variety of methods in
-    computer experiments.
+    :math:`f(x) = \frac{2 \pi T_u (H_u - H_l)}{\ln{(r/r_w)} \Big(1 +
+    \frac{2 L T_u}{\ln{(r/r_w)}r_w^2 K_w} + \frac{T_u}{T_l}\Big)}`
+
```

Comment:
Here we at least need to add the information how `x` elements map into the variables you are using. We do that for EOQ as well.